front/src/applications/stdcm/hooks/useLinkedPathSearch.ts

```diff
@@ -135,6 +135,13 @@ const useLinkedPathSearch = () => {
     }
   }, [postSearch, trainNameInput, timetableId, linkedPathDate, getExtremitiesDetails]);
 
+  useEffect(() => {
+    if (!isArrivalDateInSearchTimeWindow(linkedTrainDate, searchDatetimeWindow)) {
+      setLinkedTrainDate(selectableSlot.start);
+      launchTrainScheduleSearch();
```

Comment:
Why do you launch the search here ? I think it would be better to reset the bool hasSearchBeenLaunched and the results, what do you think ?